mono/mini/m2n-gen.cs

```diff
@@ -37,6 +37,10 @@ class Driver {
 		"VIIIIIIII",
 		"VIIIIIIIII",
 		"VIIIIIIIIII",
+		"VIIIIIIIIIII",
+		"VIIIIIIIIIIII",
+		"VIIIIIIIIIIIII",
+		"VIIIIIIIIIIIIII",
 		"I",
```

Comment:
The wasm_m2n_invoke.g.h should be generated as well, using 
csc m2n-gen.cs
mono m2n-gen.exe > wasm_m2n_invoke.g.h